package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "start": "ember serve",
```

Comment:
Official ember-cli change

Reply:
Do our docs reflect the change everywhere?

Reply:
Do our docs mention anything about package.json command aliases?

Reply:
I think we might either mention `ember s` or some variant. Good idea to `grep` and see.

Reply:
Wasn't able to check before due to replying from phone, but you were right. We had several instances where we stated to use `ember server` instead of `ember serve`.

We were also being a bit inconsistent with which to use across the docs, so I replaced all instances with `ember serve`

Note that the old command did not go away, nor is it deprecated yet. The two aliases are in for a long time now. It's just that, for some reason, did not find out which yet, they decided to change the app blueprint to specify `ember serve` as what `npm start` does.